test/appendix_a.json

```diff
@@ -69,6 +69,7 @@
     "cbor": "wkkBAAAAAAAAAAA=",
     "hex": "c249010000000000000000",
     "roundtrip": true,
+    "noncanonical": true,
```

Comment:
iirc this json was copied straight from the rfc, now this change introduces a (small) diff